Process-3 | log time - 2022-08-03 13:44:11 | log level - INFO | [main.py: line - 381 in function checkNumAds] | Формирование очереди ссылок
Process-3 | log time - 2022-08-03 13:44:11 | log level - INFO | [main.py: line - 36 in function startBrowser] | Применения прокси 192.168.0.85:3289 для запуска браузера
Process-3 | log time - 2022-08-03 13:45:00 | log level - WARNING | [connectionpool.py: line - 780 in function urlopen] | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000006313CD0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/3f969c079f25802f2c73e9f8582e58ba/window
Process-3 | log time - 2022-08-03 13:45:02 | log level - WARNING | [connectionpool.py: line - 780 in function urlopen] | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000006313DC0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/3f969c079f25802f2c73e9f8582e58ba/window
Process-3 | log time - 2022-08-03 13:45:04 | log level - WARNING | [connectionpool.py: line - 780 in function urlopen] | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000006313E20>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение')': /session/3f969c079f25802f2c73e9f8582e58ba/window
Process-3 | log time - 2022-08-03 13:45:06 | log level - INFO | [main.py: line - 36 in function startBrowser] | Применения прокси 192.168.0.85:3289 для запуска браузера
